<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IrcChannelWidget</class>
 <widget class="QWidget" name="IrcChannelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="topicLabel">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QListWidget" name="usersList">
      <property name="toolTip">
       <string>Shows the list of users</string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextBrowser" name="messages">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Shows the channel messages</string>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="E5ClearableLineEdit" name="messageEdit">
     <property name="toolTip">
      <string>Enter a message, send by pressing Return or Enter</string>
     </property>
     <property name="placeholderText">
      <string>Enter a message, send by pressing Return or Enter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5ClearableLineEdit</class>
   <extends>QLineEdit</extends>
   <header>E5Gui/E5LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>usersList</tabstop>
  <tabstop>messages</tabstop>
  <tabstop>messageEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
